src/frontend/next/package.json

```diff
@@ -15,7 +15,8 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-use": "^15.3.8",
-    "swr": "^0.4.0"
+    "swr": "^0.4.0",
```

Comment:
I think this is in master right now, so you can rebase to get it.

Reply:
Yep, I just haven't pushed the newest commits yet and stuff